eng/tools/generator/cmd/v2/common/changelogProcessor.go

```diff
@@ -78,6 +78,81 @@ func GetAllVersionTags(moduleRelativePath string) ([]string, error) {
 	return tags, nil
 }
 
+func GetAllVersionTagsV2(moduleRelativePath string, sdkRepo repo.SDKRepository) ([]string, error) {
+	arr := strings.Split(moduleRelativePath, "/")
+	log.Printf("Fetching all release tags from GitHub for RP: '%s' Package: '%s' ...", arr[len(arr)-2], arr[len(arr)-1])
+
+	// Create remote with center sdk repo if it doesn't exist
+	remoteName := "release_remote"
```

Comment:
i just recalled that we do the same thing when trying to check out the previous version's code. is it possible to consolidate the preprocessor part of them?